packages/scm/src/browser/scm-tree-model.ts

```diff
@@ -207,19 +211,19 @@ export class ScmTreeModel extends TreeModelImpl {
                 if (folderEnd - folderStart < nestingThreshold) {
```

Comment:
To me, the name "nestingThreshold" seems to indicate the depth of the tree. I don't see how the difference between two different indices in the list of resources is related to that. Shouldn't this expression involve the `level` somehow?